tests/examples/source/devfiles/nodejs/app/app.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2012-2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
```

Comment:
Does the license for this need to be apache2 to match the rest of odo?

Reply:
Removed, git cloning it instead